src/libraries/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/RuntimeTypeInfo.BindingFlags.cs

```diff
@@ -37,7 +37,7 @@ protected sealed override ConstructorInfo GetConstructorImpl(BindingFlags bindin
             }
 
             if ((bindingAttr & BindingFlags.ExactBinding) != 0)
-                return System.DefaultBinder.ExactBinding(candidates.ToArray(), types, modifiers) as ConstructorInfo;
+                return System.DefaultBinder.ExactBinding(candidates.ToArray(), types, modifiers!) as ConstructorInfo;
```

Comment:
NIT: `modifiers` better to be nullable for the the calling method `System.DefaultBinder.ExactBinding( ... )`

Reply:
Good catch, thanks!

Reply:
Good catch, thanks!